src/reanimated2/helpingHooks/Spring.js

```diff
@@ -0,0 +1,109 @@
+import { useWorklet, useSharedValue } from "../Hooks";
+import Worklet from "../Worklet";
+
+const defaultState = {
+  finished: 0,
+  velocity: 0,
+};
+
+const defaultConfig = {
+  toValue: 0,
+  damping: 10,
+  mass: 1,
+  stiffness: 100,
+  overshootClamping: 0,
+  restDisplacementThreshold: 0.001,
+  restSpeedThreshold: 0.001,
+};
+
+const workletBody = new Worklet( 
+  function (sv, state, config) {
+    'worklet';
+    const memory = Reanimated.memory(this);
```

Comment:
are you sure about this part of API? Don't you afraid about some linters or type-checks?

Reply:
I think we can export {Reanimated: memory:()=>{return {};}, ...}.